src/ip.py

```diff
@@ -114,35 +114,62 @@ def link_addr_show(argv, af, json_print, pretty_json, color, address):
         return json_dump(links, pretty_json)
 
     for l in links:
-        print("%d: %s: <%s> mtu %d status %s" % (
-            l["ifindex"],
-            colorize_ifname(color, l["ifname"]),
-            ",".join(l["flags"]),
-            l["mtu"],
-            colorize_op_state(color, l["operstate"])
-        ))
-        print(
-            "    link/" + l["link_type"] +
-            ((" " + colorize_mac(color, l["address"])) if "address" in l else "") +
-            ((" brd " + colorize_mac(color, l["broadcast"])) if "broadcast" in l else "")
-        )
-        for a in l.get("addr_info", []):
+        if brief:
+            # Brief format: interface_name STATUS ip_addresses...
+            # Interface name (left-padded to align)
```

Comment:
Let's use `rjust` or `ljust` ?

https://docs.python.org/3/library/stdtypes.html#str.rjust
https://docs.python.org/3/library/stdtypes.html#str.ljust

Btw isn't this actually right padded?